src/model.rs

```diff
@@ -1148,9 +1147,8 @@ impl Collections {
                     find_biggest_stop_areas(most_frequent_stop_areas, collections);
                 find_first_by_alphabetical_order(biggest_stop_areas)
             }
-
             let vehicle_journey_idxs = routes_to_vehicle_journeys
-                .get_corresponding_forward(&IdxSet::from_iter(std::iter::once(route_idx)));
+                .get_corresponding_forward(&std::iter::once(route_idx).collect::<IdxSet<_>>());
```

Comment:
Do you need the `IdxSet` bit since `.get_corresponding_forward()` is not generic but does take explicitely a reference of `IdxSet`?

```suggestion
                .get_corresponding_forward(&std::iter::once(route_idx).collect());
```

Reply:
Indeed! it wasn’t needed. Thanks